Apromore-Core-Components/Apromore-Manager/src/main/java/org/apromore/service/helper/UIHelper.java

```diff
@@ -252,15 +249,13 @@ public SummariesType buildLogSummaryList(String userId, Integer folderId, Intege
     }
 
     @Override
-    public UserMetadataSummaryType buildUserMetadataSummary(Group group, Usermetadata usermetadata,
-                                                            AccessType accessType) throws UserNotFoundException {
+    public UserMetadataSummaryType buildUserMetadataSummary(String username, Usermetadata usermetadata,
+                                                            AccessType accessType) {
```

Comment:
Since buildUserMetadataSummary no longer throws UserNotFoundException, what is the new behavior when an invalid username is passed?